frontend/src/pages/Billing/UpdatePlanPage.tsx

```diff
@@ -154,16 +154,6 @@ export const getNextBillingDate = (
 	}
 }
 
-export const getStripePromiseOrNull = () => {
-	const stripe_publishable_key = import.meta.env.REACT_APP_STRIPE_API_PK
-	if (stripe_publishable_key) {
-		return loadStripe(stripe_publishable_key)
-	}
-	return null
-}
-
-const stripePromiseOrNull = getStripePromiseOrNull()
```

Comment:
I was seeing Stripe taking up ~40MB of memory whenever our app loaded and tracked it down to this as the culprit. It's loading stripe as soon as the module is required since this isn't inside the component. I was still seeing it loading on page when just importing `loadStripe` as a dependency, so I ended up lazy loading it inside a `useEffect` on mount.